testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -127,6 +128,8 @@ const toSelector = (cssClass) => `.${cssClass}`;
 
 const moduleConfig = {
     beforeEach: function() {
+        viewPort($('#qunit-fixture').addClass(VIEWPORT_CLASS));
```

Comment:
shouldn't we restore viewport value in afterEach?

Reply:
well. It was not being restored in the previous version. However, using the `beforeEach` is redundant. I moved this code to QUnit.testStart. Thank you for pointing me to this